shortcircuit-api/src/main/java/com/powsybl/shortcircuit/AbstractFeederResult.java

```diff
@@ -15,13 +18,27 @@
 abstract class AbstractFeederResult implements FeederResult {
 
     private final String connectableId;
+    private final ThreeSides side;
 
     protected AbstractFeederResult(String connectableId) {
+        this(connectableId, null);
+    }
+
+    protected AbstractFeederResult(String connectableId, ThreeSides side) {
         this.connectableId = Objects.requireNonNull(connectableId);
+        this.side = side;
     }
 
     public String getConnectableId() {
         return connectableId;
     }
 
+    public ThreeSides getSide() {
```

Comment:
Could you add the missing `@Override` for the 3 methods? it should be seen as a code smell from sonar cloud, but I noticed lately it's not. Maybe because the class is abstract?
https://sonarcloud.io/organizations/powsybl-ci-github/rules?activation=true&open=java%3AS1161&q=override&qprofile=AYonwccxzxfrDz7ASaP1